From 8510e847c035f70e28ce63ff74b824c39892f602 Mon Sep 17 00:00:00 2001
From: Guillem Jover <guillem@debian.org>
Date: Tue, 4 Aug 2020 20:51:44 +0200
Subject: [PATCH] aptitude-changelog-parser: Quiesce Dpkg::Changelog parse
 warnings

The Dpkg::Changelog object gets initialized with verbose mode set to 1,
so all parsing warnings get printed on stderr, which messes up the
output when showing the changelogs from the TUI.

Starting with libdpkg-perl 1.20.6 we can change the verbose mode when
calling changelog_parse() from Dpkg::Changelog::Parse.
---
 aptitude-changelog-parser | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/aptitude-changelog-parser b/aptitude-changelog-parser
index aee1dfd1..63ff6a4f 100755
--- a/aptitude-changelog-parser
+++ b/aptitude-changelog-parser
@@ -24,4 +24,4 @@ if (scalar @ARGV == 1) {
     $opts{all} = undef;
 }
 
-print join "\n", changelog_parse(format => 'rfc822', %opts);
+print join "\n", changelog_parse(format => 'rfc822', verbose => 0, %opts);
-- 
2.33.1

